include/RAJA/DepGraphNode.hxx

```diff
@@ -92,18 +92,35 @@ public:
   DepGraphNode()
       : m_num_dep_tasks(0), m_semaphore_reload_value(0), m_semaphore_value(0)
   {
-    posix_memalign((void**)(&m_semaphore_value),
-                   _SemaphoreValueAlign_,
-                   sizeof(int));
-    *m_semaphore_value = 0;
+#ifdef _WIN32
+#if USE_ALIGNED
```

Comment:
In my world it is controlled by a Visual Studio Project. I can add it to the config - clearly a better solution.